tests/system/Debug/ExceptionsTest.php

```diff
@@ -61,4 +61,20 @@ public function testDetermineCodes(): void
         $this->assertSame([500, 1], $determineCodes(new RuntimeException('That.', 600)));
         $this->assertSame([404, 1], $determineCodes(new RuntimeException('There.', 404)));
     }
+
+    public function testRenderBacktrace(): void
+    {
+        $renderer  = self::getPrivateMethodInvoker(Exceptions::class, 'renderBacktrace');
+        $exception = new RuntimeException('This.');
+
+        $renderedBacktrace = $renderer($exception->getTrace());
+        $renderedBacktrace = explode("\n", $renderedBacktrace);
+
+        foreach ($renderedBacktrace as $trace) {
+            $this->assertMatchesRegularExpression(
+                '/^\s*\d* .+(?:\(\d+\))?: \S+(?:(?:\->|::)\S+)?\(.*\)$/',
+                $trace
+            );
+        }
+    }
```

Comment:
I don't understand what's tested.
I think we need tests to prove the changed output.

Reply:
That's true. But I added a new method. At least I don;t want it to be uncovered by tests.

Reply:
Okay, it is difficult to write tests for this.